string object_class
---
nav_msgs/Path path
string[] names